pkg/appmanager/appManager_test.go

```diff
@@ -41,10 +41,12 @@ import (
 func init() {
 	workingDir, _ := os.Getwd()
 	schemaUrl = "file://" + workingDir + "/../../schemas/bigip-virtual-server_v0.1.4.json"
+	schemaUrl5 = "file://" + workingDir + "/../../schemas/bigip-virtual-server_v0.1.5.json"
```

Comment:
What are the differences with the v0.1.5 schema - is it not backwards compatible? I think you should be able to change schemaUrl to v0.1.5 and it would still work for other tests, so no need for a new variable.

Reply:
Adding 'udp' isn't a breaking change -- so should I just add the changes to v0.1.4 of the schema?

Reply:
No, it still needs to be a new version number as it is adding some optional new thing, but we should just use v0.1.5 in unit tests then. And now that I'm thinking of it, there should also be a change in docs/README.rst to include the new schema version and the changes it brings.